### PR TITLE
chore: generate support zip for crashdump

### PR DESCRIPTION
**Pull request by frezbo**
Generate support zip on crashdump.

### CODE REVIEW

File: internal/integration/provision/provision.go [hunk 1]
```diff
@@ -142,10 +142,6 @@ func (suite *BaseSuite) TearDownSuite() {
 		// for failed tests, produce crash dump for easier debugging,
 		// as cluster is going to be torn down below
 		suite.provisioner.CrashDump(suite.ctx, suite.Cluster, os.Stderr)
-
```
Comment (frezbo) [inline internal/integration/provision/provision.go:145]: now sure if this needed to be dropped, this just seemed to gather the service logs and print to stdout, support zip  seemed a better option

File: internal/integration/provision/provision.go [hunk 2]
```diff
@@ -142,10 +142,6 @@ func (suite *BaseSuite) TearDownSuite() {
 		// for failed tests, produce crash dump for easier debugging,
 		// as cluster is going to be torn down below
 		suite.provisioner.CrashDump(suite.ctx, suite.Cluster, os.Stderr)
```
Comment (frezbo) [inline internal/integration/provision/provision.go:144]: can update this also to save to a file

**Review by smira — APPROVED** [review]
lgtm, I think Dmitry's goal is to have this even for non-failed builds

**Comment by frezbo** [comment]
> lgtm, I think Dmitry's goal is to have this even for non-failed builds

we could have another flag for intergration and cluster create that does this unconditionally

**Review by dsseng — APPROVED** [review]
Yes, my goal is to save it anyway, as for now we're testing in permissive mode and it'd be useful to be able to download audit logs anyway

**Pull request merged**